judgments/utils.py

```diff
@@ -205,7 +205,7 @@ def parse_date_parameter(params, param_name, default_to_last=False):
         day = parse_parameter_as_int(params, day_param_name, default=default_day)
 
         dt = datetime(year, month, day)
-        return dt.strftime("%Y-%m-%d")
+        return dt.strftime("%d-%m-%Y")
```

Comment:
worth adding a test for this bug fix

Reply:
Looks like we already have a failing test which just needs updating! @Donna-H let me know if you need a hand.

Reply:
@jacksonj04 updated thanks for help please review the changes